### ISSUE TITLE
Allowed only lowercase?

### ISSUE THREAD

**Issue by rwu823** [selaux/eslint-plugin-filenames#13]
```js
{
  'filenames/match-regex': ['error', '^[a-z]+$', true]
}
```

This RegExp that's mean only be allowed lowercase filename, right? But it's not working.

**Comment by selaux** [comment]
Only if that file has no named export otherwise it is:
```js
{
  'filenames/match-regex': ['error', '^[a-z]+$']
}
```

**Comment by rwu823** [comment]
@selaux 
It's still no working.

**Comment by selaux** [comment]
Can you please elaborate what is not working? Are files not reported as errors? How does the directory structure look etc.

**Comment by rwu823** [comment]
```
root/
├── Dev/
└── tasks/
```

The `Dev` dir or filename is passed in the rule.

**Comment by selaux** [comment]
Do other rules apply to the tasks directory? (Sorry, didn't see the response)

**Comment by selaux** [comment]
Closed due to inactivity.
